tester/src/test/scala/spinal/tester/scalatest/SpinalSimWishboneSlaveFactoryTester.scala

```diff
@@ -33,25 +35,34 @@ class SpinalSimWishboneSlaveFactoryTester extends SpinalAnyFunSuite{
       if(dut.io.bus.config.useLOCK) dut.io.bus.LOCK #= false
       if(dut.io.bus.config.isPipelined) dut.io.bus.STALL #= false
 
-      dut.io.bus.ACK #= false
-      dut.io.bus.DAT_MISO #= 0
+      dut.clockDomain.waitSampling()
 
       val scoreboard = ScoreboardInOrder[WishboneTransaction]()
-      val driver = new WishboneDriver(dut.io.bus, dut.clockDomain)
-      // val monitor = WishboneMonitor(dut.io.bus, dut.clockDomain){ bus =>
-      //   if(bus.WE.toBoolean)
-      //     scoreboard.pushRef(WishboneTransaction.sampleAsSlave(bus))
-      //   else
-      //     scoreboard.pushDut(WishboneTransaction.sampleAsMaster(bus))
-      // }
-
-      for(repeat <- 0 until 1000){
-        driver.drive(for(x <- 1 to 10) yield WishboneTransaction(address=10).randomizeData(dut.io.bus.config.dataWidth),true)
-        dut.clockDomain.waitSampling(10)
-        driver.drive(for(x <- 1 to 10) yield WishboneTransaction(address=10),false)
-        dut.clockDomain.waitSampling(10)
+      scoreboard.ref ++= List(
```

Comment:
same as above, please use `scoreboard.pushRef(...)` and add a `scoreboard.checkEmptiness()` at the end of the test